streams/shared/src/main/scala/zio/stream/experimental/Take.scala

```diff
@@ -16,61 +16,104 @@
 
 package zio.stream.experimental
 
-import zio.{ Cause, IO, ZIO }
+import zio.{ Cause, Chunk, ZIO }
 
 /**
- * A `Take[E, B, A]` represents a single `take` from a queue modeling a stream of
+ * A `Take[E, A]` represents a single `take` from a queue modeling a stream of
  * values. A `Take` may be a failure cause `Cause[E]`, an element value `A`
- * or an end-of-stream marker `B`.
+ * or an end-of-stream marker.
  */
-sealed trait Take[+E, +B, +A] extends Product with Serializable { self =>
```

Comment:
TBH I'm not sure `Take` is still useful so far, it was just weird in the `effectAsync` that I could just write `unsafeRun(output.offer(k))`. But I could do that, what do you think @iravid ?

Reply:
Let's get rid of it and use `Exit[Option[E], Chunk[A]]` instead.